ios/RNCClipboard.m

```diff
@@ -28,4 +28,19 @@ - (dispatch_queue_t)methodQueue
   resolve((clipboard.string ? : @""));
 }
 
+RCT_EXPORT_METHOD(hasString:(RCTPromiseResolveBlock)resolve
+                  reject:(__unused RCTPromiseRejectBlock)reject)
+{
+  bool stringPresent = false;
```

Comment:
nit: `BOOL` should be preferred in ObjC, with `YES` or `NO` as values.

See https://stackoverflow.com/questions/541289/objective-c-bool-vs-bool